packages/node/src/client.ts

```diff
@@ -138,6 +147,46 @@ export class NodeClient extends BaseClient<NodeClientOptions> {
     );
   }
 
+  /**
+   * Create a cron monitor check in and send it to Sentry.
+   *
+   * @param checkIn An object that describes a check in.
+   * @param upsertMonitorConfig An optional object that describes a monitor config. Use this if you want
+   * to create a monitor automatically when sending a check in.
+   */
+  public captureCheckIn(checkIn: CheckIn, monitorConfig?: MonitorConfig): void {
+    if (!this._isEnabled()) {
+      __DEBUG_BUILD__ && logger.warn('SDK not enabled, will not capture checkin.');
+      return;
+    }
+
+    const options = this.getOptions();
+    const { release, environment, tunnel } = options;
+
+    const serializedCheckIn: SerializedCheckIn = dropUndefinedKeys({
```

Comment:
I am wondering if `dropUndefinedKeys` is still necessary (at all in the codebase) after https://github.com/getsentry/sentry-javascript/pull/8017 🤔

Reply:
Kept it just to be safe, but not sure.

Reply:
Ended up removing in 1cdfd9fd81bbc8dd227994e1f8a9cf3056634380